src/backend/dx11/src/device.rs

```diff
@@ -185,13 +185,13 @@ impl Device {
 
         // See [`shader::introspect_spirv_vertex_semantic_remapping`] for details of why this is needed.
         let semantics: Vec<_> = attributes.iter().map(|attrib| {
-            match vertex_semantic_remapping.get(&attrib.location).unwrap() {
```

Comment:
in what case would the remapping *not* contain the attrib location? It seems like `unwrap()` could be well suited here

Reply:
If a pipeline describes `0 => Float4, 1 => Float4`, but then the vertex shader only has:
```glsl
layout(location = 0) in vec4 input;
```
then there is no entry for location 1, causing a panic.

Reply:
ok, sounds good